Podcast.xcodeproj/project.pbxproj

```diff
@@ -66,7 +66,11 @@
 		44E471091E63D85300CC7101 /* Int+.swift in Sources */ = {isa = PBXBuildFile; fileRef = 44E471081E63D85300CC7101 /* Int+.swift */; };
 		6512F2601FB11D070055727F /* FeedSeriesTableViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6512F25F1FB11D070055727F /* FeedSeriesTableViewCell.swift */; };
 		651B832F1FB10EFE001B3630 /* FeedEpisodeTableViewCell.swift in Sources */ = {isa = PBXBuildFile; fileRef = 651B832E1FB10EFE001B3630 /* FeedEpisodeTableViewCell.swift */; };
+		6DA94AB92038895500EBA146 /* DownloadManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 6DA94AB82038895500EBA146 /* DownloadManager.swift */; };
```

Comment:
Not relevant to this file .. but you forgot to add downloading from bookmarks 
<img width="257" alt="screen shot 2018-03-01 at 12 03 00 am" src="https://user-images.githubusercontent.com/14966713/36827851-0a54c6cc-1ce4-11e8-9e17-db8dfd9ba2d3.png">

Reply:
No it's in the action sheet. There is no "Downloaded" icon for that cell

Reply:
ahh i see ok!